sub_circuit_identification/Makefile

```diff
@@ -0,0 +1,29 @@
+#
+
+#library_graphs:	basic_library/
+#	python3 ./src/read_library.py --dir basic_library
+
+INPUT=/dataVolume/topology_input
+SRC=/DEMO/src
+
+Results/${DESIGN}_lef.sh:
+
+library_graphs:	/DEMO/basic_library
+	python3 $(SRC)/read_library.py --dir /DEMO/basic_library
+
+circuit_graphs/${DESIGN}.yaml: ${INPUT}/${DESIGN}.sp library_graphs
+	python3 $(SRC)/read_netlist.py --dir ${INPUT} -f `basename $< .sp`.sp --subckt ${DESIGN} --flat 0
+
+Results/${DESIGN}.p:	circuit_graphs/${DESIGN}.yaml
+	python3 $(SRC)/match_graph.py
+
+Results/${DESIGN}_lef.sh:	Results/${DESIGN}.p LEF
+	python3 $(SRC)/write_verilog_lef.py -U_cap 12 -U_mos 12
+	python3 $(SRC)/check_const.py
```

Comment:
I think "check_const.py" should take ${DESIGN} as an argument somehow. That corrected some errors. (Rebase with master to see.)

Reply:
I have some conflicts with Master.  I wasn't really ready for review, I just got this working last night, but it automatically requests review.
I should demonstrate how to use it locally as it is quite cool.  It resolves the problem (for me) of fixing a step in the flow locally, then rerunning quickly.  I can show you a much faster workflow for local that makes more sense.   I also now have an argument why the extra complexity of docker is worth it as this system scales --- it will encapsulate complexity needed now in some of the modules environments.
I got it to the point where it claims it is posting the PNGs but I can't see an artifact location.  So still debugging that.
I struggled mightily to get a method for WSL.   It will work automatically for WSL2.  I learned a lot about how WSL works and can look into where things _really_ go when you use a bind mount.

One thing that is driving me nuts:  I can't figure out what Codacy wants from me regarding the README.md.  It wants a space added but also deleted.  I just don't understand and I've given up on something that should be trivial.

I think for running locally you need to set ALIGN_HOME if you have not.  I'll check with a fresh shell when I get in.